### PR TITLE
DDCNL-3943 | play2.8 upgrade

### PR DESCRIPTION
**Pull request by shikkhasingh**

### CODE REVIEW

**Comment by struong-man** [comment]
This failed to compile btw, check the PR builder

File: project/plugins.sbt [hunk 1]
```diff
@@ -11,9 +11,10 @@ val playPlugin =
    sys.env.getOrElse("PLAY_VERSION", "2.6") match {
     case "2.6" => "com.typesafe.play" % "sbt-plugin" % "2.6.20"
     case "2.7" => "com.typesafe.play" % "sbt-plugin" % "2.7.4"
+    case "2.8" => "com.typesafe.play" % "sbt-plugin" % "2.8.0"
```
Comment (struong-man) [inline project/plugins.sbt]: Lets take on 2.8.7

**Review by struong-man — CHANGES_REQUESTED** [review]

**Review by struong-man — DISMISSED** [review]

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/master/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/master/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/master/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/master/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Review by vimalsuresh — APPROVED** [review]

**Pull request merged**